ci/infra/testrunner/tests/test_skuba_addon_upgrade.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+import yaml
+import semantic_version
+import re
+from io import StringIO
+
+
+def decrease_version(version_str):
+    v = semantic_version.Version(version_str)
+    if v.patch > 0:
+        v.patch -= 1
+    elif v.minor > 0:
+        v.minor -= 1
+    elif v.major > 1:
+        v.major -= 1
+    else:
+        v.major = 0
+        v.minor = 1
+    return str(v)
+
+def test_addons_upgrade(deployment, kubectl, skuba):
+    all_fine = re.compile(
+        r'Congratulations! Addons for \d+\.\d+\.\d+ are already at the latest version available'
+    )
+
+    out = skuba.addon_upgrade('plan')
```

Comment:
can we have this separated into it's own test? similar to `node upgrade plan` and `node upgrade apply`